addons/event/models/event_mail.py

```diff
@@ -262,7 +262,8 @@ def schedule_communications(self, autocommit=False):
         schedulers = self.search([
             ('event_id.active', '=', True),
             ('mail_done', '=', False),
-            ('scheduled_date', '<=', fields.Datetime.now())
+            ('scheduled_date', '<=', fields.Datetime.now()),
+            ('event_id.stage_id', '!=', self.env.ref('event.event_stage_cancelled').id)
```

Comment:
Can you add a comma at the end of this line so that the next addition to the domain won't change the history of your line.